datasets/csv/csv.py

```diff
@@ -11,10 +10,11 @@
 @dataclass
 class CsvConfig(nlp.BuilderConfig):
     """BuilderConfig for CSV."""
+
     skip_rows: int = 0
     header_as_column_names: bool = True
     delimiter: str = ","
-    quote_char: str = "\""
+    quote_char: str = '"'
```

Comment:
@thomwolf - black did this automatically - is that ok?

Reply:
Yes!